cirq/ops/pauli_string.py

```diff
@@ -115,6 +130,24 @@ def __rmul__(self, other):
             return PauliString(self._qubit_pauli_map, self._coefficient * other)
         return NotImplemented
 
+    def __add__(self, other):
+        from cirq.ops.linear_combinations import PauliSum
+        if isinstance(other, (int, float, complex)):
+            return PauliSum.from_pauli_strings(
+                [self, PauliString(coefficient=other)])
+        elif isinstance(other, PauliString):
+            return PauliSum.from_pauli_strings([self, other])
+        return other + self
+
+    def __sub__(self, other):
+        from cirq.ops.linear_combinations import PauliSum
```

Comment:
Do we also need `__add__` and `__sub__` on Pauli gates X/Y/Z?

Reply:
I added a test with single-qubit pauli gates which I expected to fail so then I could implement these, but lo and behold `SingleQubitPauliStringGateOperation` inherits from `PauliString` so it works already